doc/Makefile

```diff
@@ -40,9 +40,10 @@ help:
 .PHONY: clean
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	rm -rf $(BUILDDIR)/
```

Comment:
Yes. I tested the build process after I've changed the `make clean` and it worked normally.